skulpin-renderer/src/skia_support.rs

```diff
@@ -92,7 +92,16 @@ pub struct VkSkiaSurface {
 
 impl VkSkiaSurface {
     pub fn get_image_from_skia_texture(texture: &skia_safe::gpu::BackendTexture) -> vk::Image {
-        unsafe { std::mem::transmute(texture.vulkan_image_info().unwrap().image.as_ref().unwrap()) }
+        unsafe {
+            std::mem::transmute(
+                texture
+                    .vulkan_image_info()
+                    .unwrap()
+                    .image()
```

Comment:
This is the actual change: `.image` to `.image()`.  
When it was one line, `cargo fmt` reported an error in the CI.